openslides_backend/action/actions/committee/import_meeting.py

```diff
@@ -0,0 +1,241 @@
+import time
+from typing import Any, Dict, Iterable, Optional, Tuple
+
+from ....models.base import model_registry
+from ....models.fields import (
+    BaseTemplateField,
+    GenericRelationField,
+    GenericRelationListField,
+    RelationField,
+    RelationListField,
+)
+from ....models.models import Committee
+from ....shared.exceptions import ActionException, MissingPermission
+from ....shared.filters import FilterOperator
+from ....shared.interfaces.event import EventType
+from ....shared.interfaces.write_request import WriteRequest
+from ....shared.patterns import KEYSEPARATOR, Collection, FullQualifiedId
+from ...action import Action
+from ...util.crypto import get_random_string
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData, ActionResults
+
+
+@register_action("committee.import_meeting")
+class CommitteeImportMeeting(Action):
+    """
+    Action to import a meeting.
+    """
+
+    model = Committee()
+    schema = DefaultSchema(Committee()).get_update_schema(
+        additional_required_fields={"meeting_json": {"type": "object"}}
+    )
+
+    def perform(
+        self, action_data: ActionData, user_id: int, internal: bool = False
+    ) -> Tuple[Optional[WriteRequest], Optional[ActionResults]]:
+        """
+        Simplified entrypoint to perform the action.
+        """
+        self.user_id = user_id
+        self.index = 0
+        for instance in action_data:
```

Comment:
Since I do not think we need it, you can just make this a `SingularAction` and skip the whole index and action data handling since there is only one action data element.

Reply:
Use the SingularAction Mixin.